### ISSUE TITLE
Focus obscured behind nav bar

### ISSUE THREAD

**Issue by hanshillen** [adaptlearning/adapt-contrib-core#635]
If a focusable element receives focus while it's is positioned behind the navigation bar, it remains hidden.  

This is disorienting for sighted keyboard and screen magnifier users, and fails both WCAG SC 2.4.11
Focus Not Obscured (Minimum) and SC 2.4.12 Focus Not Obscured (Enhanced).

Note: this is specifically about elements receiving focus through tabbing and standard DOM `focus()` calls, not about moving focus using `router.navigateToElement()`. 

This can be solved by setting `scroll-padding-top`:

```less
body {
  scroll-padding-top: var(--adapt-navigation-height);
}
```

This will scroll any element into view if it receives focus while (partially) behind the nav bar.
